docs/populating-configuration.md

```diff
@@ -18,17 +18,62 @@ to test things out.
 
 >NOTE: You could pass dummy values to make sure that the secrets files are created.
 
-## Setting up OCM token
+## Setting up OCM tokens
 
-[OpenShift Cluster Managament](https://api.openshift.com/) is utilised for the management of Data plane clusters. 
-To be able to talk to this service, retrieve your ocm-offline-token from https://qaprodauth.cloud.redhat.com/openshift/token 
-and save and run the following command
+[OpenShift Cluster Management](https://api.openshift.com/) in Fleet Manager is
+used for mainly two purposes:
+* By Fleet Manager, to perform management of Data Plane clusters
+* By the users, to interact with the Fleet Manager API: OpenShift Cluster
+  Management (OCM) tokens are used as an Authentication mechanism to interact
+  with the Fleet Manager API
+
+different kind of OCM tokens need to be generated and/or configured for each
+purpose. Keep reading below on how to do so.
+
+### Interacting with the Fleet Manager API
+
+The Fleet Manager API requires OCM Bearer tokens for authentication. The
+token is passed as an `Authorization` HTTP header value.
+
+To be able to perform requests to the Fleet Manager API, you first need
+to get an OCM token by requesting it to the OCM API. This can be done in an
```

Comment:
Based on what I've said on https://github.com/bf2fc6cc711aee1a0c2a/fleet-manager-go-template/pull/10/files#r803549522 I suggest we rephrase this to 
```suggestion
To be able to perform requests to the Fleet Manager API, you first need
to get a jwt token from sso.redhat.com, authentication server used by default. This can be done in an
```

Reply:
Reworded a little bit